src/Randomizer.Data/WorldData/Regions/Region.cs

```diff
@@ -106,23 +108,23 @@ private bool MatchesItemPlacementRule(Item item)
         {
             var rule = Config.ItemPlacementRule;
             if (rule == ItemPlacementRule.Anywhere
-                || (!item.Progression && !item.IsKey && !item.IsKeycard && !item.IsBigKey)
```

Comment:
Why did you remove the parentheses here?

Reply:
Huh. I definitely don't remember doing this, and it doesn't seem like something I would do. I've noticed visual studio has started to recommend removing of unnecessary parenthesis, so I wonder if it did it automatically when I moved files? These are superfluous parenthesis, but they really help readability IMO.

Reply:
Either we need to update the .editorconfig or something's broken. Again.